lib/schemas/HAR/entry.json

```diff
@@ -36,18 +36,6 @@
     "timings": {
       "$ref": "timings.json#"
     },
-    "serverIPAddress": {
-      "type": "string",
-      "oneOf": [
-        { "format": "ipv4" },
-        { "format": "ipv6" },
-        { "maxLength": 0 },
-        { "format": "ipv6brackets" }
-      ]
-    },
-    "connection": {
```

Comment:
This is not mentioned in the specification, is removal intended?

Reply:
Yes, skipping `connection` is intended, will update the specification to reflect this